26 feature stylesheet

Nayomekaia opened the pull request:
### What I did
Created a stylesheet for the project.

_We created a styleguide in figma i made the stylesheet based on this exact styleguide_ 
<img width="676" height="431" alt="Scherm­afbeelding 2025-10-01 om 15 27 11" src="https://github.com/user-attachments/assets/efe82799-acce-4dfa-8378-fefcaca9b163" />

_what i did in short_
- Added typography, color variables, and button styles as the design foundation.
- Linked the stylesheet in +layout.svelte so it’s applied globally.
- Imported fonts so they can be used across the project.
- Built a small HTML example to show the differences between headings, paragraphs, and buttons.
- Added hover and focus states for buttons to ensure consistency throughout the project.

### Visual examples
<img width="711" height="774" alt="Scherm­afbeelding 2025-10-01 om 15 42 49" src="https://github.com/user-attachments/assets/079320ab-936e-4591-a6fe-f205c784a67c" />


## Hoverstates + focus state 

**figma**
<img width="754" height="494" alt="Scherm­afbeelding 2025-10-01 om 15 36 12" src="https://github.com/user-attachments/assets/45a1f46c-cc7d-4bdf-993a-93015f089354" />


**Stylesheet**

_hover state_

https://github.com/user-attachments/assets/0012c853-1ce0-4a11-91b1-f5009cbaf887

_focus state_

https://github.com/user-attachments/assets/481b50d7-3061-4854-9440-f20ca434b1e7

### Feedback : 
- [ ] does it look the same as we designed in figma?
- [ ] am i repeating things in my css that could be written shorter or cleaner?
- [ ] are the hover and focus states good or do we want to change them?
- [ ] are the colors, fonts and spacing  good?
- [ ] are the files loading correctly (stylesheet, fonts)?
- [ ] is there something missing that i should still add?

ArmanVD reviewed (commented):
Looks very solid overall! I just made a few comments on items that i think need to stay in the layout.svelte file. Also 1 change in the lineheight in stylesheet.css

For your feedback requests:

- It looks exactly the same as we designed in Figma!! Very good
- A few things, @ambersr mentioned this in her review
- Hover and focus states are perfect, no change needed!
- Colours, fonts, and spacing are exactly as we designed so they are good, yes.
- Unfortunately we do not have a live preview to test this, but it looks like they are set-up correctly
- NO!

ambersr reviewed (commented):
### Feedback:
- The stylesheet looks the exact same as the styleguide, perfect!
- The heading elements code could be written shorter (see my comment)
- The hover and focus state colors are good. Also for accessiblity this is a good addition.
- I also see you added my previous review, good addition!

Nayomekaia commented:
Thank you for your feedback!
I’ve added the removed items back to +layout.svelte, corrected the stylesheet value from 18 to 1.8, and shortened the headings.

ArmanVD reviewed (approved):
Great! Now that you've implemented the feedback I think we can merge it to the dev branch!
